test/example/example.pb.go

```diff
@@ -305,7 +308,8 @@ func (m *B) Unmarshal(data []byte) error {
 			if postIndex > l {
 				return io.ErrUnexpectedEOF
 			}
-			m.G = append(m.G, github_com_gogo_protobuf_test_custom.Uint128{})
```

Comment:
I'm not sure if it's an old change or not, but this is the code generated by `make all` on my machine (OS X 10.10.3, protoc 2.6.1 installed via homebrew). I can revert it.